src/core/server/base.js

```diff
@@ -13,19 +13,21 @@ import { match } from 'react-router';
 import { ReduxAsyncConnect, loadOnServer } from 'redux-async-connect';
 
 import WebpackIsomorphicTools from 'webpack-isomorphic-tools';
-import WebpackIsomorphicToolsConfig from 'config/webpack-isomorphic-tools';
+import WebpackIsomorphicToolsConfig from 'webpack-isomorphic-tools-config';
 
 import config from 'config';
 import { setJWT } from 'core/actions';
 
+const env = config.util.getEnv('NODE_ENV');
+const isDeployed = ['stage', 'dev', 'production'].indexOf(env) > -1;
```

Comment:
`isDeployed` is supposed to represent that it's running on a server e.g. `stage`, `-dev` or `production` I'm not quite sure what you're aiming for with wasBuilt?